### ISSUE TITLE
No module named 'requests_toolbelt'

### ISSUE THREAD

**Issue by wie0wt8i9guo** [requests/toolbelt#324]
Hello,
I am running python 3.8 in a venv and working on comms with API.  I have no issues with requests, but for some reason requests_toolbelt cannot be found.  This appears to be a common issue.  I changed permissions of installation directory (which worked for one individual... but not me).

Is there any way that you can make the installation more robust?from requests_toolbelt.multipart.encoder import MultipartEncoder

ModuleNotFoundError: No module named 'requests_toolbelt'

**Comment by sigmavirus24** [comment]
This module installs without issue for all twine users. I don't know how you're installing this, but it sounds like a problem with your set-up, not with this module

**Comment by wie0wt8i9guo** [comment]
Thank you @sigmavirus24 for your reply.  I will continue to look into how my system could cause an issue.

**Comment by wie0wt8i9guo** [comment]
A bit of progress... I am running VS Code and there are 3 Python interpreters available.
     Python 3.8 64 (which I have used since creation of this venv)
     Python 3.6.9  
     Python 2.7.17

It appears that PIP is associated with Python 3.6.9 because I do not receive this error when I run using that interpreter.  I will do a bit more research to find out how to separate 3.8 from the others and re-install.

I hope this will save someone else time in the future.

**Comment by wie0wt8i9guo** [comment]
FIXED... I needed to refresh my list of interpreters.  When I had installed Python 3.8, it was only in this virtual environment.  I clicked the refresh button on my list of interpreters...
![image](https://user-images.githubusercontent.com/98130596/150445012-7054465c-b02f-4e8f-a535-454458d4012b.png)

... and I noticed a 3.8.0 version specific to this workspace / venv

![image](https://user-images.githubusercontent.com/98130596/150445151-dcc27a0d-c763-4b21-b478-39a0010fce25.png)

I selected the 3.8 version related to this workspace / venv and I was able to run my code without receiving this error.

I hope it helps someone!!
